addons/l10n_it_edi/models/account_move_send.py

```diff
@@ -100,12 +105,19 @@ def _link_invoice_documents(self, invoices_data):
         # EXTENDS 'account'
         super()._link_invoice_documents(invoices_data)
 
-        attachments_vals = [
-            invoice_data.get('l10n_it_edi_values')
-            for invoice_data in invoices_data.values()
-            if invoice_data.get('l10n_it_edi_values')
-        ]
-        if attachments_vals:
-            attachments = self.env['ir.attachment'].sudo().create(attachments_vals)
-            res_ids = attachments.mapped('res_id')
-            self.env['account.move'].browse(res_ids).invalidate_recordset(fnames=['l10n_it_edi_attachment_id', 'l10n_it_edi_attachment_file'])
+        move_ids_to_names = {}
+        for move, data in invoices_data.items():
+            if values := data.get('l10n_it_edi_values'):
+                move.l10n_it_edi_attachment_file = base64.b64encode(values['raw'])
+                move.l10n_it_edi_attachment_name = values['name']
```

Comment:
```suggestion
                move.l10n_it_edi_attachment_file = base64.b64encode(values['raw'])
                move.l10n_it_edi_attachment_file.name = values['name']
                move.l10n_it_edi_attachment_name = values['name']
```
Can't we simply do this?
But it makes me wonder why we even need the field `l10n_it_edi_attachment_name` at all?
Can't we just get it from the attachment?